src/test/kotlin/sirius/web/http/CorsTest.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.http
+
+import io.netty.handler.codec.http.HttpHeaderNames
+import io.netty.handler.codec.http.HttpMethod
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.kernel.BaseSpecification
+import sirius.kernel.SiriusExtension
+import java.net.HttpURLConnection
+import java.net.URL
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@ExtendWith(SiriusExtension::class)
+class CORSTest {
+    @Test
+    fun `expect 'Access-Control-Allow-Origin' for requests with 'origin'`() {
+        // Allow us to set the Origin: header...
+        System.setProperty("sun.net.http.allowRestrictedHeaders", "true")
+        val c = URL("http://localhost:9999/system/ok").openConnection() as HttpURLConnection
```

Comment:
Constructor URL is deprecated, also in AssetDispatcherTest.kt -> use URI.toURL()